linkage-monitor/src/test/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitorTest.java

```diff
@@ -207,7 +217,7 @@ public void testGenerateMessageForFixedError() {
     assertEquals(
         "The following problems in the baseline no longer appear in the snapshot:\n"
```

Comment:
don't we want problems in the snapshot that do not appear in the baseline?

Reply:
Yes, we want that.

This test case testGenerateMessageForFixedError is to show the opposite: showing problems in the baseline that do not appear in the snapshot. Such problems are just recorded and they do not throw error.